lib/apr/views/seller_slack_view.ex

```diff
@@ -7,7 +7,10 @@ defmodule Apr.Views.InvoiceSlackView do
 
     cond do
       routing_key =~ "merchantaccount" ->
-        merchant_account_message(event, partner_data)
+        case event["verb"] do
+          "external_account_restricted_soon" -> external_account_restricted_soon_message(event, partner_data)
+          _ -> merchant_account_message(event, partner_data)
+        end
 
       routing_key =~ "invoicetransaction" ->
```

Comment:
_I think_ this event can be retired as the only publisher (a.k.a. [lewitt-api](https://github.com/search?q=InvoiceTransaction+org%3Aartsy&type=code)) has been retired. Not blocking for this PR tho.